include/zephyr/net/lwm2m.h

```diff
@@ -215,6 +216,25 @@ struct lwm2m_ctx {
 	uint8_t validate_buf[CONFIG_LWM2M_ENGINE_VALIDATION_BUFFER_SIZE];
 };
 
+/**
+ * LwM2M Time series data structure
+ */
+struct lwm2m_time_series_elem {
+	/* Cached data Unix timestamp */
+	time_t t;
+	union {
+		uint8_t u8;
+		uint16_t u16;
+		uint32_t u32;
+		uint32_t u64;
+		int8_t i8;
+		int8_t i16;
+		int8_t i32;
+		int8_t i64;
```

Comment:
Ditto, those should use respective signed int types?

Reply:
ah let me fix.